src/effects/VST3/VST3UIValidator.cpp

```diff
@@ -176,6 +186,9 @@ bool VST3UIValidator::UpdateUI()
    if (mPlainUI != nullptr)
       mPlainUI->ReloadParameters();
 
+   //Write to main...
+   mAccess.Flush();
```

Comment:
Is this flush needed to fix the bug?

It used to be that there was no Flush, and every Get did what Flush does.  I thought this caused too slow updating of meters, at least for AudioUnits, because of frequent UpdateUI calls. so I made Flush a different thing called less often.

Reply:
Flush is called frequently in LV2 now without observable performance hit. Looking into debugger I can say it is called more frequently than UpdateUI does.

Reply:
In VST3 UpdateUI called only on preset change